core/src/archive/zip.rs

```diff
@@ -13,7 +13,7 @@ pub fn extract<Tracker: ProgressTracker>(
 ) -> Result<()> {
     let source = File::open(target)?;
     if !target_dir.exists() {
-        fs::create_dir(target_dir)?;
+        fs::create_dir_all(target_dir)?;
```

Comment:
What's the reason for this change?

Reply:
Now that we are putting the releases for the games an extra level deep in the data dir, the directory to extract the releases to may not have already been created.

For example, at startup we ensure the data dir `~/.SpicyLauncher` exists, and that was where we would extract the releases.

Now, with multiple games, though, we need to extract to `~/.SpicyLauncher/jumpy` or another game-specific folder which may not exist yet.

Alternatively we could make sure directories for each game exist before hand.